modular_skyrat/modules/bsrpd/code/bsrpd.dm

```diff
@@ -15,6 +15,7 @@
 	var/current_capacity = BSRPD_CAPACITY_MAX
 	var/ranged_use_cost = BSRPD_CAPACITY_USE
 	var/in_use = FALSE
+	var/toggle = FALSE
```

Comment:
`toggle` is really vague and is begging to collide with some future feature, can this be `allow_ranged` or something?